device/iot-device-samples/send-event/src/main/java/samples/com/microsoft/azure/sdk/iot/SendEvent.java

```diff
@@ -80,92 +80,58 @@ else if (status == IotHubConnectionStatus.CONNECTED)
     public static void main(String[] args)
             throws IOException, URISyntaxException
     {
+
+        InputParameters params = new InputParameters(args);
+
         System.out.println("Starting...");
         System.out.println("Beginning setup.");
 
-        if (args.length <= 1 || args.length >= 5)
-        {
-            System.out.format(
-                    "Expected 2 or 3 arguments but received: %d.\n"
-                            + "The program should be called with the following args: \n"
-                            + "1. [Device connection string] - String containing Hostname, Device Id & Device Key in one of the following formats: HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key> or HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>;GatewayHostName=<gateway> \n"
-                            + "2. [number of requests to send]\n"
-                            + "3. (mqtt | https | amqps | amqps_ws | mqtt_ws)\n"
-                            + "4. (optional) path to certificate to enable one-way authentication over ssl \n",
-                    args.length);
-            return;
-        }
-
-        String connString = args[0];
+        String connString = params.getConnectionString();
         int numRequests;
-        String pathToCertificate = null;
+        String pathToCertificate;
         try
         {
-            numRequests = Integer.parseInt(args[1]);
+            numRequests = Integer.parseInt(params.getNumberOfRequests());
         }
         catch (NumberFormatException e)
         {
             System.out.format(
                     "Could not parse the number of requests to send. "
-                            + "Expected an int but received:\n%s.\n", args[1]);
+                            + "Expected an int but received:[" + params.getNumberOfRequests() + "]");
```

Comment:
Added a space to this.

```suggestion
                            + "Expected an int but received: [" + params.getNumberOfRequests() + "]");
```

Reply:
> will correct this